packages/beacon-node/src/network/gossip/gossipsub.ts

```diff
@@ -135,6 +135,10 @@ export class Eth2Gossipsub extends GossipSub {
       // if this is false, only publish to mesh peers. If there is not enough GOSSIP_D mesh peers,
       // publish to some more topic peers to make sure we always publish to at least GOSSIP_D peers
       floodPublish: !opts?.disableFloodPublish,
+      // Only send IDONTWANT messages if the message size is larger than this
+      // This should be large enough to not send IDONTWANT for "small" messages
+      // See https://github.com/ChainSafe/lodestar/pull/7077#issuecomment-2383679472
+      idontwantMinDataSize: 16829,
```

Comment:
regarding https://github.com/ChainSafe/lodestar/pull/7077#issuecomment-2383679472, isn't the `maxSize` inflated because the list limits are way to high but in reality can't happen because those are just aggregates over committees while the limit is the block attestation max.

Anyhow doesn't matter much as long as this is smaller than block / blob size

Reply:
Yeah, its way higher than needed, but it should suffice for now.